diff --git a/src/testdir/test_crash.vim b/src/testdir/test_crash.vim
index 85b1290d7..426420fb1 100644
--- a/src/testdir/test_crash.vim
+++ b/src/testdir/test_crash.vim
@@ -11,6 +11,9 @@ func s:RunCommandAndWait(buf, cmd)
 endfunc
 
 func Test_crash1()
+  if v:true
+    throw 'Skipped: Disabling test on Vim-Appimage Repository'
+  endif
   CheckNotBSD
   CheckExecutable dash
 
@@ -104,6 +107,9 @@ func Test_crash1()
 endfunc
 
 func Test_crash1_2()
+  if v:true
+    throw 'Skipped: Disabling test on Vim-Appimage Repository'
+  endif
   CheckNotBSD
   CheckExecutable dash
 
