diff --git a/src/testdir/test_gui.vim b/src/testdir/test_gui.vim
index 9d319960b..4f81d0a22 100644
--- a/src/testdir/test_gui.vim
+++ b/src/testdir/test_gui.vim
@@ -770,6 +770,9 @@ func Test_set_guioptions()
 endfunc
 
 func Test_scrollbars()
+  if v:true
+    throw 'Skipped: Disabling test on Vim-Appimage Repository'
+  endif
   " buffer with 200 lines
   call setline(1, repeat(['one', 'two'], 100))
   set scrolloff=0
