diff --git a/src/testdir/Makefile b/src/testdir/Makefile
index 2fb329aa1..70479632d 100644
--- a/src/testdir/Makefile
+++ b/src/testdir/Makefile
@@ -12,7 +12,7 @@ SCRIPTSOURCE = ../../runtime
 # Comment out this line to see the verbose output of tests.
 #
 # Catches SwapExists to avoid hanging at the ATTENTION prompt.
-REDIR_TEST_TO_NULL = --cmd 'au SwapExists * let v:swapchoice = "e"' > /dev/null
+REDIR_TEST_TO_NULL = --cmd 'au SwapExists * let v:swapchoice = "e"'
 
 # Uncomment this line to use valgrind for memory leaks and extra warnings.
 #   The output goes into a file "valgrind.testN"
diff --git a/src/testdir/test_options.vim b/src/testdir/test_options.vim
index 74f8d4c55..59054226f 100644
--- a/src/testdir/test_options.vim
+++ b/src/testdir/test_options.vim
@@ -594,7 +594,7 @@ func Test_backupskip()
   let after =<< trim [CODE]
       let bsklist = split(&backupskip, ',')
       call assert_equal(uniq(copy(bsklist)), bsklist)
-      call writefile(['errors:'] + v:errors, 'Xtestout')
+      call writefile(['errors:'] + v:errors, expand("$PWD") .. '/Xtestout')
       qall
   [CODE]
   call writefile(after, 'Xafter')
@@ -874,7 +874,7 @@ func Test_opt_default_cdpath()
   CheckFeature file_in_path
   let after =<< trim [CODE]
     call assert_equal(',/path/to/dir1,/path/to/dir2', &cdpath)
-    call writefile(v:errors, 'Xtestout')
+    call writefile(v:errors, expand("$PWD") .. '/Xtestout')
     qall
   [CODE]
   if has('unix')
